main.go

```diff
@@ -265,3 +307,21 @@ func appendIfNotExists(base []vulnerabilityreport.VulnerabilityLabel, items []vu
 
 	return result
 }
+
+func appendIfNotExistsCIS(base []ciskubebenchreport.ReportLabel, items []ciskubebenchreport.ReportLabel) []ciskubebenchreport.ReportLabel {
```

Comment:
Yeah you're right this is unfortunate. Let's try to remember to clean this up with the next big update.

Reply:
Noted, I agree we can change it on the next major